service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipClaimService.java

```diff
@@ -146,6 +149,7 @@ public MasProcessingObject markAsRfd(MasProcessingObject payload) {
 
     try {
       bipApiService.updateClaimStatus(claimId, ClaimStatus.RFD);
+      saveToDbService.updateRfdFlag(String.valueOf(claimId), true);
```

Comment:
I see there are two places to update the claim status to ClaimStatus.RFD. Is it intended to save the info only in this place (not for completeProcessing)?